client/app/components/about-modal/about-modal.controller.js

```diff
@@ -0,0 +1,35 @@
+(function() {
+  'use strict';
+    
+  angular.module('app.components')
+    .directive('aboutModal', function() {
+      return {
+        templateUrl: "app/components/about-modal/about-modal.html",
+        controller: ModalController,
+        controllerAs: "vm",
+      };
+    });
+  /** @ngInject */
+  function ModalController(ServerInfo) {
+    var vm = this;
+    vm.additionalInfo = "";
+    vm.copyright = __("Copyright (c) 2016 ManageIQ. Sponsored by Red Hat Inc.");
+    vm.imgAlt = "ManageIQ logo";
```

Comment:
Verified that the last version of the changes works with skinning (after https://github.com/ManageIQ/manageiq-ui-self_service/pull/192 gets in so that skinning actually works again :)).

@Fryguy I'm not sure about the copyright info, if we want the skins to be able to override that as well, this will need changing, but I suggest we do that in a separate PR, ..dependencies.